workers/social/lib/social/models/group/groupdata.coffee

```diff
@@ -15,5 +19,83 @@ module.exports = class JGroupData extends Module
         type      : String
         validate  : require('../name').validateName
         set       : (value) -> value.toLowerCase()
+      payload     : Object
```

Comment:
@gokmen i had to rename this field, bongo was overriding it. i tried to refactor bongo but that required a lot of effort and reverted my changes.